### PR TITLE
added lastmod to sponsorship

### PR DESCRIPTION
**Pull request by yungalyx**

### CODE REVIEW

**Review by rmehri01 — COMMENTED** [review]
Now that I think of it, shouldn't sponsors have a rank too?

**Comment by yungalyx** [comment]
@rmehri01 you're right, ill ask spons + design

File: nw-firestore-schema.ts [hunk 1]
```diff
@@ -2,13 +2,16 @@ interface Sponsor {
   name: "apple";
   url: "apple.com";
   imgURL: "link-to-image";
+  lastmod: Date().toLocaleString(),
+  teir: "platinum"
```
Comment (Ciuffi) [inline nw-firestore-schema.ts]: Typo in teir, also could you make this a type, so its easier to understand what the options are?

aka
```typescript
type Tier = "gold" | "silver" | "platinum"
````

and then in the sponsors interface:

```typescript
interface Sponsor {
tier: Tier = "gold";
}
```

File: nw-firestore-schema.ts [hunk 2]
```diff
@@ -2,13 +2,16 @@ interface Sponsor {
...
 }
 
 interface Event {
   name: "event name";
   description: "this is an event";
   imgURL?: "link-to-event-image";
   order: "ordering of the event";
+
```
Comment (Ciuffi) [inline nw-firestore-schema.ts:16]: Should remove the extra empty space, leads to ugly code.

**Review by Ciuffi — CHANGES_REQUESTED** [review]
Sorry for the nitpicky comments, this just makes the documentation easier to understand.

**Review by Ciuffi — APPROVED** [review]
fixed

**Pull request merged**